src/utils/media.ts

```diff
@@ -0,0 +1,29 @@
+export function MediaException(message) {
+  this.message = message;
+  this.name = 'MediaException';
+}
+
+/**
+ * Get the media element when the component loads or is updated
+ */
+export function getMedia(element: string) {
+  let mediaSource = document.getElementById(element);
```

Comment:
global reference here is *not* ok.

Pass in the `<host/>` element and use it's query selector API